src/mqt/predictor/reward.py

```diff
@@ -16,7 +17,8 @@
 
 logger = logging.getLogger("mqt-predictor")
 
-figure_of_merit = Literal["expected_fidelity", "critical_depth"]
+FIGURES_OF_MERIT = ["expected_fidelity", "critical_depth", "expected_success_probability"]
+figure_of_merit = Literal["expected_fidelity", "critical_depth", "expected_success_probability"]
```

Comment:
```suggestion
figure_of_merit = Literal[FIGURES_OF_MERIT]
```

By this, we could avoid the duplication.

Reply:
Yeah I tried this too but then there is this error `Type arguments for "Literal" must be None, a literal value (int, bool, str, or bytes), or an enum valuePylance`.
`Literal[*FIGURES_OF_MERIT]` is not possible either.

Reply:
Lines 20 and 21 seem to be very similar. Is there no way we can get rid of one of them? This kind of code duplication should be avoided.

Reply:
Well, at least there is no way that Copilot or I could find yet.